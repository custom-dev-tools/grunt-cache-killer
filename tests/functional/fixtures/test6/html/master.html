<!DOCTYPE html>
<html lang="en">
<head>
    <!-- Metadata -->
    <meta charset="UTF-8">

    <!-- Title -->
    <title>Test HTML Template - Master</title>

    <!-- Stylesheets -->
    <link rel="stylesheet" href="../css/style.min.css">
</head>
<body>
    <h1 id="heading">Test HTML Template - Master</h1>
</body>
</html>