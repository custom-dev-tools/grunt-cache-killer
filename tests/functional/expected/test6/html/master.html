<!DOCTYPE html>
<html lang="en">
<head>
    <!-- Metadata -->
    <meta charset="UTF-8">

    <!-- Title -->
    <title>Test HTML Template - Master</title>

    <!-- Stylesheets -->
    <link rel="stylesheet" href="../css/style.9063da07.min.css">
</head>
<body>
    <h1 id="heading">Test HTML Template - Master</h1>
</body>
</html>